tests/e2e/specs/plugin-reset.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * WordPress dependencies
+ */
+import { deactivatePlugin, activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Reset plugin', () => {
+	beforeAll( async() => {
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await activatePlugin( 'google-site-kit' );
+	} );
+
+	afterAll( async() => {
+		await deactivatePlugin( 'e2e-tests-auth-plugin' );
+		await deactivatePlugin( 'google-site-kit' );
+	} );
+
+	beforeEach( async() => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-settings' );
+		await page.waitForSelector( '.mdc-tab-scroller__scroll-content' );
+
+		// Click on Admin Settings Tab.
+		await page.$$eval( '.mdc-tab-scroller__scroll-content button', tabs => {
+			Array.apply( null, tabs )
+				.find( element => 'Admin Settings' === element.textContent )
+				.click();
+		} );
+
+		// Click on Reset Site Kit.
+		await page.evaluate( () => {
```

Comment:
Likewise:

```js
await expect( page ).toClick( '.googlesitekit-cta-link', { text: 'Reset Site Kit' } );
```

Although I'd argue that clicking on the reset button should be part of each test to make it clear that the step needs confirmation.